### PR TITLE
Add links and quotes OSS related - move things

### PR DESCRIPTION
**Pull request by smellems**
Attempt to make sense of all the things that are in here.  6.1 is the point about Open Standards and OSS.  Can we mention OSS specifically in 6.1?  I think bringing embrace leading practices down under 6.2 with use common platforms would be good.  I started to move things around with that in mind.

Still figuring out how to best contribute to this

Related to #106

### CODE REVIEW

**Comment by pjackson28** [comment]
The title for 6.1 came directly from the latest version of the Digital Standards (same with the titles for 6, 6.2, 6.3 and 6.4) so that can't change without the Digital Standards themselves changing first. Everything else can change though.

Overall, what can and can't change, are the following:

**Can't change:**
- Standard titles - Comes directly from the Digital Standards so would have to be changed there first
- Titles for first four guidelines of each Standard (i.e., x.1, x.2, x.3, x.4) - Comes directly from the Digital Standards so would have to be changed there first
- Any OneGC Architectural Checklist/Built It Right items - Comes directly from the OneGC Architectural Checklist/Build It Right so would have to be changed there first

**Can change, but needs to be done globally (with agreement from the other areas):**
- Common section names such as "Checklist", "Implementation guides", "Similar resources", "Guidelines", "Related guidelines"

**Can change:**
- Anything else (although subject to review by the reviewers for that section/topic)

File: en/6-use-open-standards-solutions.md [hunk 1]
```diff
@@ -73,8 +115,17 @@ Using open standards and common government platforms helps you to:
 
 **[TODO: Add/revise checklist items]**
 
+- For all software acquisition, open source software must be actively and fairly considered alongside proprietary ones **(SSC Draft Position on OSS (GC))**
+- Where there is no significant overall (full-lifecycle) cost difference between open source and closed-source products, open source will be selected on the basis of its additional inherent flexibility **(SSC Draft Position on OSS (GC))**
+- OSS support models and options must be evaluated and selected **(SSC Draft Position on OSS (GC))**
+- All software, including OSS, must have a defined and managed lifecycle that is supported by a robust toolset (software acquisition and distribution guidelines) **(SSC Draft Position on OSS (GC))**
+
```
Comment (pjackson28) [inline en/6-use-open-standards-solutions.md]: This blank line will result in a second list. Is that intentional? Are you trying to group the items separately?

File: en/6-use-open-standards-solutions.md [hunk 2]
```diff
@@ -32,12 +32,54 @@ altLangPage: 6-utiliser-normes-solutions-ouvertes
...
 
+Give equal consideration to open source software when you choose technology.
+
+Your technology project or programme could benefit from:
+
+- solving common problems with readily available open source technology
+- more time and resource for customised solutions to solve the rare or unique problems
+- lower implementation and running costs
+
+Be aware that open source software is not completely free, so take into account the total cost of migrating, including exit and transition costs.
```
Comment (pjackson28) [inline en/6-use-open-standards-solutions.md]: Is "take into account the total cost of migrating" addressed in the checklist? It sounds like an action that should be taken so would be appropriate for a checklist item (if a checklist item covering this aspect doesn't already exist).

File: en/6-use-open-standards-solutions.md [hunk 3]
```diff
@@ -32,12 +32,54 @@ altLangPage: 6-utiliser-normes-solutions-ouvertes
...
 <section class="dpgn-section-guideline">
 
-## 6.1 Leverage open standards and embrace leading practices
+## 6.1 Leverage open standards, use open source software and embrace leading practices
 
 <div class="dpgn-section-intro-guideline">
 
 **[TODO: Add/revise introductory text]**
 
+Give equal consideration to open source software when you choose technology.
```
Comment (pjackson28) [inline en/6-use-open-standards-solutions.md]: This sounds like an action that should be taken. Is it already covered adequately in the checklist items?

File: en/6-use-open-standards-solutions.md [hunk 4]
```diff
@@ -32,12 +32,54 @@ altLangPage: 6-utiliser-normes-solutions-ouvertes
...
+
+- entail proven benefits, namely with respect to quality, safety of information, variety of functionalities and the pooling of solutions;
+- contribute to the interoperability, reusability and interchangeability of solutions;
+- contribute to data resilience;
+- contribute to eliminating vendor lock-in;
+- constitute an industry of their own and must be assessed on merit, like every other industry.
+
+The city acknowledges that, as a public administration body, it must promote free competition in terms of software and hardware purchases.
+
+- Principle 1
```
Comment (pjackson28) [inline en/6-use-open-standards-solutions.md]: Principles 1, 2 and 3 seem like they could be actions that should be taken. Are there checklist items (in the Checklist section) that already cover these actions?

File: en/6-use-open-standards-solutions.md (hunk 2 above)
Comment (gcharest) [reply en/6-use-open-standards-solutions.md]: That's a good question that we will need to submit to the Checklist team (I'm understanding here the Build-it-right checklist?)

Comment (pjackson28) [reply en/6-use-open-standards-solutions.md]: Build It Right is currently only a small subset of the checklist items, so wouldn't apply in this case. For changes to non-Build It Right checklist items (and adding checklist items), they would just go to the regular reviewers for the section (so the same reviewers as for the introduction).

If you are actually looking to change a Build It Right checklist item, then it should be changed through Build It Right first before changing it in the Digital Playbook.

Comment (gcharest) [reply en/6-use-open-standards-solutions.md]: Sorry @pjackson28 , I was confused as I thought we were referring to the three layers of the EA.
We can definitely update the checklists here first.

**Comment by smellems** [comment]
@pjackson28 thanks for reviewing this.  I think I would like to take a different approach in another PR.  I would like to start writing the statements and checklists for the GC and remove all quotes from others that are only there for reference (instead of adding more).  If were keeping it that's good otherwise having the link in Similar resources is good. What do you think?

**Comment by pjackson28** [comment]
@smellems That sounds like a good plan. The quoted material from other jurisdictions was included both a placeholder and to provide inspiration. That content can be changed, removed and added to as needed as it wasn't intended to be the final product.

I do agree that it is good to continue to reference the source documents in the similar resources section as we do use that content to generate the [comparison table](https://canada-ca.github.io/digital-playbook-guide-numerique/views/comparison/en/gc-digital-playbook-comparison-table.html).

**Pull request closed without merging**